lib/HapiSpool.ts

```diff
@@ -45,12 +45,13 @@ export class HapiSpool extends ServerSpool {
   configure () {
     this.app.config.set('web.server', 'hapi')
     this.app.config.set('web.routes.files.relativeTo', this.app.config.get('main.paths.root'))
+    this.serverConfig = this.app.config.get('web.options');
```

Comment:
This might pass tests if you remove the semicolon

Reply:
Unit tests are failing during the CI Build. The last PR was merged without CI running unit tests and then CI was turned on after